optimize/components/userguide/collections-dashboards-reports.md

```diff
@@ -24,10 +24,6 @@ To [create a dashboard](./creating-dashboards.md) or [report](./creating-reports
 
 ## User permissions
 
-:::note
-Adding user groups to collections is currently only available in Camunda 7.
-:::
-
 By default, if you create a collection, only you can access the collection and the contents within. To share a collection with other users, add them to the collection.
 
 ![users and user groups](./img/users.png)
```

Comment:
We also don't have user groups in C8, might be worth cleaning up while you're here